src/DataTransformer/ChainOutputDataTransformer.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\DataTransformer;
+
+/**
+ * Transforms an Output to a Resource object.
+ *
+ * @author Antoine Bluchet <soyuka@gmail.com>
+ */
+final class ChainOutputDataTransformer implements DataTransformerInterface
+{
+    private $transformers;
+
+    public function __construct(\Traversable $transformers)
+    {
+        $this->transformers = $transformers;
+    }
+
+    /**
+     * Creates a serialization context from a Request.
+     */
+    public function transform($object, array $context = [])
```

Comment:
```suggestion
    public function transform(object $object, array $context = [])
```

Reply:
7.2 only for object typing, I wanted to type it, also the return type

Reply:
Ok! At least at the PHPDoc then.